zebra.py

```diff
@@ -0,0 +1,50 @@
+from ophyd import (Component as Cpt, EpicsSignal, EpicsSignalRO, Device)
+
+
+class ZebraPCBase(Device):
+    sel = Cpt(EpicsSignal, 'SEL')
+    start = Cpt(EpicsSignal, 'START')
+    width = Cpt(EpicsSignal, 'WID')
+    step = Cpt(EpicsSignal, 'STEP')
+
+
+class ZebraPCGate(ZebraPCBase):
+    num_gates = Cpt(EpicsSignal, 'NGATE')
+
+
+class ZebraPCPulse(ZebraPCBase):
+    max = Cpt(EpicsSignal, 'MAX')
+    delay = Cpt(EpicsSignal, 'DLY')
+    status = Cpt(EpicsSignalRO, 'INP.STA')
+
+
+class ZebraPositionCompare(Device):
+    arm_status = Cpt(EpicsSignalRO, 'ARM_INP.STA')
+    download_count = Cpt(EpicsSignalRO, 'NUM_DOWN')
+    disarm = Cpt(EpicsSignal, 'DISARM')
+    encoder = Cpt(EpicsSignal, 'ENC')
+    enc_x = Cpt(EpicsSignal, 'ENC1')
+    enc_y = Cpt(EpicsSignal, 'ENC2')
+    enc_z = Cpt(EpicsSignal, 'ENC3')
+    enc_omega = Cpt(EpicsSignal, 'ENC4')
+    encoder = Cpt(EpicsSignal, 'ENC')
```

Comment:
It looks like `encoder` was defined the same way twice in this class (lines 25 and 30).

Reply:
thanks, again! turns out I had not pushed the comment through when we talked at the meeting.